src/vma/dev/ring_tap.h

```diff
@@ -69,8 +69,9 @@ class ring_tap : public ring_slave
 
 	virtual int modify_ratelimit(struct vma_rate_limit_t &rate_limit) { NOT_IN_USE(rate_limit); return 0; }
 	void inc_cq_moderation_stats(size_t sz_data) { NOT_IN_USE(sz_data); }
-	virtual int get_max_tx_inline() { return 0; }
 	virtual uint32_t get_underly_qpn() { return -1; }
+	virtual uint32_t get_max_inline_data() { return 0; }
+	virtual uint32_t get_max_send_sge() { return 1; }
```

Comment:
Why do we want to disable it for ring tap   ?

Reply:
probably compilation isssue